packages/main/src/Input.js

```diff
@@ -407,6 +433,13 @@ class Input extends UI5Element {
 			}
 		}
 
+		if (!this.firstRendering && !this.Suggestions && this._getPopover() && this.hasValueStateMessage) {
+			this.toggle(this.shouldDisplayOnlyValueStateMessage);
+			if (this._getPopover().contentDOM) {
+				this._getPopover().contentDOM.style.display = "none";
```

Comment:
Do not directly access the DOM unless absolutely necessary. Change some state of the component that reflects in the "get styles()" and uses the template to update the styles. In order words, don't change DOM directly, only through the template. The reason is that you introduce hidden state that will be hard to track in the future

Reply:
Done.